pkg/reconciler/instances/connectivityproxy/command.go

```diff
@@ -134,5 +135,22 @@ func (a *CommandActions) Remove(context *service.ActionContext) error {
 	if err != nil {
 		return errors.Wrap(err, "Error during removal")
 	}
+
+	return a.RemoveIstioSecrets(context)
+}
+
+func (a *CommandActions) RemoveIstioSecrets(context *service.ActionContext) error {
```

Comment:
I would suggest  make this method as package private since it is the only place when it is executed.

Reply:
Maybe the name can be a little bit more clear, like RemoveIstioSecretsFromShoot, but such name is loo long?

Reply:
Personally, I think the `RemoveIstioSecretsFromShoot` is a bit too long. We are performing the deletion on the shoot, so we can figure out this function is linked to the shoot.